docs/hf_hub.rst

```diff
@@ -111,3 +111,43 @@ returns the model output for a given input. It can be used as::
 
 In the above code snippet, ``res`` will be a :class:`numpy.ndarray` containing
 the model's output.
+
+.. _hf_hub_gradio:
+..
+   TODO: replace gradio link once gradio provides object.inv
+Easily build user interfaces to your scikit-learn models
+--------------------------------------------------------
+`gradio <https://gradio.app/>`__ is a python library that lets you create interfaces on your model.
+It has a class called :class:`Interface` that lets you create application
```

Comment:
`:class:Interface` and `:func:gr.Interface.launch()` still don't work (sorry for wonky quote but GH won't let me put backticks inside backticks D-:). For the users, it would be helpful to replace them for the time being with links to https://gradio.app/docs/#interface and https://gradio.app/docs/#launch

Reply:
fixed them